keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -28,7 +28,7 @@
 BOUNDING_BOXES = "bounding_boxes"
 KEYPOINTS = "keypoints"
 RAGGED_BOUNDING_BOXES = "ragged_bounding_boxes"
-SEGMENTATION_MASK = "segmentation_mask"
+SEGMENTATION_MASK = "segmentation_masks"
```

Comment:
SEGMENTATION_MASK[S]?

Reply:
Done. Made it plural for consistency with the other labels